src/azure-cli/setup.py

```diff
@@ -71,7 +71,7 @@
     'azure-mgmt-botservice~=0.2.0',
     'azure-mgmt-cdn==4.1.0rc1',
     'azure-mgmt-cognitiveservices~=5.0.0',
-    'azure-mgmt-compute~=11.0',
+    'azure-mgmt-compute~=12.0',
```

Comment:
thanks for the full live test regression. @qwordy  +1 for live test framework

Reply:
I'm running the full live test for VM in my local. And hope there will be no new failure.